docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -2,61 +2,162 @@
-** *Project Authoring* perspective contains:
-*** The *Project Explorer* view with the overview of the repository structure, and information about available resources, such as business process definitions, form definitions, and others.
-*** The editor area on the right of the *Project Explorer* view, where the respective editor appears when a resource is opened.
-*** The *Messages* view with validation messages.
-** *Contributors* perspective enables you to view the number of commits sorted by the organizational unit, repository, author, and other criteria.
-** *Artifact Repository* perspective contains a list of jars which can be added as dependencies. The available operations in this perspective are upload/download artifact and open (view) the `pom.xml` file.
+Click the house icon on the top left of any screen to return to this page.
+
+[NOTE]
+====
+Depending on your role, you may not see all of the perspectives listed. 
+====
+
+.Design
+The Design group contains two perspectives:
+
+* Projects
++
+The Projects perspective enables you to import or  create Business Central projects. This is where you do your process design work. It contains a number of example projects.
++
+This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+
+* Dashboards
++
+The Dashboards perspective is the BAM component of {PRODUCT}. Use it to create custom perspectives for your projects. On the Dashboards perspective, you can create a new group for the *Main* menu and add the perspectives that you create to that group. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+
+.DevOps
+The DevOps group contains two perspectives:
+
+* Deployments
+
+[IMPORTANT]
+====
+The Deployments perspective is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs), might
+not be functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+The Deployments perspective enables you to provision projects on OpenShift. In this release, it is hidden by default. Perform the following steps to enable it:
+
+. Go to *Settings* → *Roles*.
+. Select the *admin* role.
+. In the Permissions section, click the arrow next to Perspectives.
+. Click the trash icon next to Deployments. The DevOps → *Deployments* link is now available on the Home page.
+
+* Execution Servers
+The Execution Servers perspective contains a list of the deployed Intelligent Process Server templates and containers associated with the templates. This perspective is visible to users with the following roles:
```

Comment:
In the html output,  the sentence "The Execution Servers perspective contains a list of the deployed ..." is not appearing in a different line.